src/controllers/update/update.interrupt.card.controller.ts

```diff
@@ -0,0 +1,18 @@
+import { NextFunction, Request, Response } from "express";
+
+import { logger } from "../../utils/logger";
+import * as config from "../../config";
+
+export const get = (req: Request, res: Response, next: NextFunction) => {
+  try {
+    logger.debugRequest(req, `${req.method} ${req.route.path}`);
+
+    return res.render(config.UPDATE_INTERRUPT_CARD_PAGE, {
+      backLinkUrl: "",
```

Comment:
Is the back link the use paper page? Assume it's interrupting the forward from that ...

Reply:
Added the backlink now for the secure update filter page.